### PR TITLE
Several fixes about chrome application window (size and state)

### PR DESCRIPTION
**Pull request by mandrichenko**
Tested in Chrome 55.0.2883.95 (64-bit) on:
- Windows 10 Pro x64
- macOS Sierra 10.12.2

1. Window state restored with previous saved window state (fullscreen or normal).
Prior to this, the application window does not open in full screen. Now app stored last window state.

2. After closing the game, window restored in previous state.
Prior to this, after closing the game the application window opened in "normal mode", even if user entered in "fullscreen mode". 

3. Application window startup size changed to 16:9 aspect ratio = 960x540.
Cosmetic changes, because major resolution is 1920x1080 = 16:9

### CODE REVIEW

File: static/js/messages.js [hunk 1]
```diff
@@ -30,7 +30,6 @@ function handleMessage(msg) {
                     });
                 });
                 showApps(api);
-                chrome.app.window.current().restore();
```
Comment (raidancampbell) [inline static/js/messages.js]: Why was this line removed?  With this line removed, after exiting a stream my window is always fullscreen.  With this line in place, the previous dimensions of the window are restored

File: static/js/background.js [hunk 2]
```diff
@@ -1,8 +1,45 @@
...
+
+function createWindow(state) {
     chrome.app.window.create('index.html', {
-        state: "normal",
-        bounds: { 
-    		width: 850, height: 500
-    	}
+        state: state,
+        bounds: {
+            width: 960,
```
Comment (raidancampbell) [inline static/js/background.js]: ++ for correcting this.  I specifically remember sitting at my computer and loosely saying "yeah, that looks like a decent size"

**Comment by raidancampbell** [comment]
I finally got around to testing this code on OSX 10.11 with Chrome 55.0.2883.95.

    `Window state restored with previous saved window state (fullscreen or normal).  
    Prior to this, the application window does not open in full screen.
    Now app stored last window state.`

 - If I launch through `chrome://extensions`, I don't see any difference between your branch and `master`.  I'll assume that Chrome is just doing weird things here.
- If I launch directly from OSX, I can verify that this works as you described

```
After closing the game, window restored in previous state.
Prior to this, after closing the game the application window opened in "normal mode", 
even if user entered in "fullscreen mode".
```
- I cannot replicate your fixed behavior with your branch as-is.  [My comment](https://github.com/moonlight-stream/moonlight-chrome/pull/158/files/5ebcc58ae31b4ad3c7ead44d51e2be82d19a32cf#diff-87126795dc662dbdc0d0755d3d098746) on L34 of `messages.js` stands.  With your change, exiting a stream results in moonlight still being in fullscreen, even if I entered from a `normal` window size.  If I add that line back in, I see behavior as you described: `normal`->stream->stop->`normal` and `fullscreen`->stream->stop->`fullscreen`.

```
Application window startup size changed to 16:9 aspect ratio = 960x540.
Cosmetic changes, because major resolution is 1920x1080 = 16:9
```
- Nice catch. Looks good to me

**Comment by mandrichenko** [comment]
> - If I launch through chrome://extensions, I don't see any difference between your branch and master. I'll assume that Chrome is just doing weird things here. 
> - If I launch directly from OSX, I can verify that this works as you described

You're right. This applies to the launch from `chrome://extensions` or `chrome://apps`, when I use default constructor `chrome.window.create('index.html', { state: 'normal' })` for some reasons it is not work properly. I created and tested on a new project - the same behavior.
That was fixed by manually invoking function to restore window: `window.restore()`.

> I cannot replicate your fixed behavior with your branch as-is. My comment on L34 of messages.js stands. With your change, exiting a stream results in moonlight still being in fullscreen, even if I entered from a normal window size. If I add that line back in, I see behavior as you described: normal->stream->stop->normal and fullscreen->stream->stop->fullscreen.

Now it works as expected.

Tested on Windows 10 and macOS 10.12.2 with Chrome 55.0.2883.95.

**Comment by raidancampbell** [comment]
Tested and confirmed working.
